README.md

```diff
@@ -1,1981 +1,97 @@
+Amazon Sumerian Hosts is an experimental open source project that aims to make it easy to create interactive animated 3D characters that can be rendered on the Web and leverage AWS Services such as [Amazon Polly](https://aws.amazon.com/polly/). 
+It defines a Javascript API for managing animations, synthesizing and playing speech with Amazon Polly, and generating lipsync animation at runtime in sync with Polly generated audio. Amazon Sumerian Hosts is divided up into three packages:
+* [Amazon-Sumerian-Hosts-Core](packages/amazon-sumerian-hosts-core) is the core API that can be extended to support the Web rendering engine of your choice
+* [Amazon-Sumerian-Hosts-Three](packages/amazon-sumerian-hosts-three) is an example of an integration of the core library with [three.js](https://threejs.org/)
+* [Amazon-Sumerian-Hosts-Babylon](packages/amazon-sumerian-hosts-babylon) is an example of an integration of the core library with [Babylon.js](https://www.babylonjs.com/)
 
-You can clone the repository to obtain glTF character animation assets tailored to work well with the Host API. You can use them as-is, modify them in DCC software such as [Blender](https://www.blender.org/) or use them as a guide to develop your own 3D assets.
+You can clone the repository to obtain glTF character animation assets (located in the [examples](packages/amazon-sumerian-hosts-babylon/examples/assets/glTF/)) tailored to work well with the Host API. You can use them as-is, modify them in DCC software such as [Blender](https://www.blender.org/) or use them as a guide to develop your own 3D assets.
 
-The easiest way to start using the API is to include the build file that corresponds with the rendering engine you are using. See the [Getting Started](#Getting-Started) guide for a walkthrough using this method and the [API Documentation](https://aws-samples.github.io/amazon-sumerian-hosts/) for more detailed information on the classes and methods available. Amazon Sumerian Hosts is a published [npm](https://www.npmjs.com/) package, so alternatively you can install in an existing Node.js project by running `npm install --save-dev amazon-sumerian-hosts`. If you'd like to pull the gitub repository and create your own build, see [Building the Package](#Building-the-Package) for prerequisites and instructions on how to do that.
+The easiest way to start using the API is to include the build file that corresponds with the rendering engine you are using. See the [Getting Started](#Getting-Started) guide for a walkthrough using this method and the [API Documentation](https://aws-samples.github.io/amazon-sumerian-hosts/) for more detailed information on the classes and methods available. Amazon Sumerian Hosts has all three of its packages published to [npm](https://www.npmjs.com/), so alternatively you can install in an existing Node.js project by running `npm install --save-dev @amazon-sumerian-hosts/core @amazon-sumerian-hosts/three @amazon-sumerian-hosts/babylon`.

```

Comment:
The "Getting Started" link is broken and the wording doesn't quite make sense now that the "Getting Started" guide has been split into separate Babylon and Three "Gettting Started" guides. In this main readme, I would call out each Getting Started guide individually and link to both of them.